### PR TITLE
Add deploy cluster-proxy addon scenario

### PR DESCRIPTION
**Pull request by ycyaoxdu**
Signed-off-by: ycyaoxdu <yaoyuchen0626@163.com>

https://github.com/open-cluster-management-io/OCM/issues/17 
tested on my local machine, and i used kind to create clusters.

### CODE REVIEW

**Comment by qiujian16** [comment]
/assign @yue9944882 

With the portforwad feature added, not sure if we still need lb installed.

File: solutions/deploy-cluster-proxy-addon/README.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
+# Deploy cluster-proxy addon
+ 
+## Prerequisite
+
+Set up the dev environment in your local machine following [setup dev environment](../setup-dev-environment).
+
+## Support provisioning LoadBalancer typed service for kind
```
Comment (yue9944882) [inline solutions/deploy-cluster-proxy-addon/README.md:7]: the latest release of cluster-proxy doesnt require LoadBalancer anymore. can we clarify that this step is optional?

**Review by yue9944882 — COMMENTED** [review]
my apologies for the delay!

File: solutions/deploy-cluster-proxy-addon/README.md (hunk 1 above)
Comment (ycyaoxdu) [reply solutions/deploy-cluster-proxy-addon/README.md:7]: alright.😁

**Comment by ycyaoxdu** [comment]
> my apologies for the delay!

never mind!

**Comment by qiujian16** [comment]
thanks together with https://github.com/open-cluster-management-io/OCM/pull/20, I am thinking a full scenario that:

1. user enable cluster-proxy and managed-sa addon
2. user create managed sa
2. user can get the status of the cluster from /healthy (clusteradm proxy)

**Pull request closed without merging**